src/containers/mobile/landing/hero/hero-styles.module.scss

```diff
@@ -10,6 +10,10 @@
   position: relative;
   width: 100vw;
 
+  @media (orientation: landscape) {
```

Comment:
Also here, should we add a mobile class so we don't affect other screens?

Reply:
Is inside a mobile component